docker-compose-test-tools.yml

```diff
@@ -33,14 +33,17 @@ services:
     container_name: edgex-app-service-configurable
     hostname: app-service-configurable
     environment:
+      EDGEX_SECURITY_SECRET_STORE: "false"
```

Comment:
This is no longer needed. It was a work around until the real fix of `don't attempt to create Secret Client if optional SecretStore config is missing`. That fix is in now.